CanDIG/katsu
# Sonchau/create db script

## What's new:
- Added a new script to rebuild the metadata database if deleted.
- Cleaned up unnecessary hostnames.

### How to test:
- Check out this branch
- Run test-integration.
- To test the new script:
  - Delete the metadata database from Postgres.
  - Execute create_db.sh.
